src/common/turingHelper.js

```diff
@@ -96,6 +102,23 @@ class TuringHelper {
     });
 
     getProxyAccount = (parachainId, address) => getProxyAccount(this.api, parachainId, address);
+
+    getProxies = async (address) => getProxies(this.api, address);
+
+    getFeePerSecond = async (assetId) => {
+        const { additional: { feePerSecond } } = (await this.api.query.assetRegistry.metadata(assetId)).toJSON();
```

Comment:
Confirmed this with Nikhil that this is the fee_per_second we are using to calculate SDN fees of automation.

Reply:
Yes, Nikil agrees with it.